tests/network-tests/index.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs');
```

Comment:
When and how should this script be used.
index.js is not very informative is its not really the root file for the network-tests package. Did you replace this file with `utils/api-scripts/src/fork-off.ts` perhaps ?